app/src/main/res/layout/activity_splash.xml

```diff
@@ -5,28 +5,31 @@
     android:layout_height="match_parent"
     android:theme="@style/SplashTheme">
 
-    <androidx.constraintlayout.widget.Guideline
-        android:id="@+id/start"
+    <LinearLayout
```

Comment:
Nesting this inside `ConstraintLayout` is unnecessary.

Reply:
Ok, I have removed ConstraintLayout and set LinearLayout as root layout.